app/models/family.rb

```diff
@@ -5,7 +5,7 @@ class Family < ApplicationRecord
 
   belongs_to :tank, required: false
 
-  def name
-    "Male: #{male.id} / Female: #{female.id}"
-  end
+  # def name
+  # "Male: #{male.id} / Female: #{female.id}"
+  # end
```

Comment:
Can we just delete this method? It will be stored in the git history if we ever need it again.